packages/react-instantsearch-hooks-server/src/getServerState.tsx

```diff
@@ -86,26 +124,33 @@ function waitForResults(search: InstantSearch) {
   });
 }
 
+/**
+ * Recurse over all child indices
+ *
+ * @param indexWidget initial index widget
+ * @param callback gets called for each index widget
+ * @returns void
```

Comment:
The function description is useful but we don't add annotations in the other functions, because they're explicit enough to be used internally, and reduces the maintenance cost. I think it's fair to remove them. Wdyt?

```suggestion
```